include/z64player.h

```diff
@@ -473,44 +474,50 @@ typedef enum PlayerModelGroup {
     /* 15 */ PLAYER_MODELGROUP_MAX
 } PlayerModelGroup;
 
-typedef enum PlayerEyeIndex {
+typedef struct PlayerFaceIndices {
+    /* 0x0 */ u8 eyeIndex;
+    /* 0x1 */ u8 mouthIndex;
+} PlayerFaceIndices; // size = 0x2
```

Comment:
In OoT, they opted to use an enum and a 2D array instead of a struct. i.e.
```c
typedef enum PlayerFacePart {
    /* 0 */ PLAYER_FACEPART_EYES,
    /* 1 */ PLAYER_FACEPART_MOUTH,
    /* 2 */ PLAYER_FACEPART_MAX
} PlayerFacePart;
```
I like the struct version better, but open to opinions

Reply:
I agree the struct looks nicer.